Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:latest as build-dev
```

Comment:
using `latest` since this is really just for the CI, it will be nice to remain current.